src/store/TransactionStore.ts

```diff
@@ -141,7 +141,7 @@ class TransactionStore {
             const XCHspendsList = await Wallet.generateXCHSpendList({
                 puzzleReveal,
                 amount: '0',
-                memo,
+                memo: '', // fee no need memo
```

Comment:
no need => memo is unnecessary for fee

Reply:
done
[45a030a](https://github.com/hashgreen/hoogii-wallet/pull/19/commits/45a030a02e34176c57a35cae449ddc3bd94a74d0)